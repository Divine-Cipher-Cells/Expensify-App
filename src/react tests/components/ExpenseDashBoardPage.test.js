import ExpenseDashBoardPage from "../../components/ExpenseDashBoardPage";
import React from "react";
import { shallow } from "enzyme";


test("Expense dash board component",() => {
    const wrapper = shallow(<ExpenseDashBoardPage />);
    expect(wrapper).toMatchSnapshot();
    });