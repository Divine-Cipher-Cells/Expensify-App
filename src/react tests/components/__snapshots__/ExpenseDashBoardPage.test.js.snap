// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expense dash board component 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
